src/cares_wrap.cc

```diff
@@ -912,14 +883,12 @@ void AfterGetAddrInfo(uv_getaddrinfo_t* req, int status, struct addrinfo* res) {
   HandleScope handle_scope(env->isolate());
   Context::Scope context_scope(env->context());
 
-  Local<Value> argv[] = {
-    Integer::New(env->isolate(), status),
-    Null(env->isolate())
-  };
+  Local<Value> argv[] = {Integer::New(env->isolate(), status),
```

Comment:
It's good to see there's one argument per line now, but IMHO the previous format in this and other similar cases are more readable.